content/zh/blog/_posts/2018-08-29-kubernetes-testing-ci-automating-contributor-experience.md

```diff
@@ -0,0 +1,239 @@
+<!--
+---
+layout: blog
+title:  'The Machines Can Do the Work, a Story of Kubernetes Testing, CI, and Automating the Contributor Experience'
+date:   2018-08-29
+---
+
+**Author**: Aaron Crickenberger (Google) and Benjamin Elder (Google)
+-->
+
+---
+layout: blog
+title: '机器可以完成这项工作，一个关于 kubernetes 测试、CI 和自动化贡献者体验的故事'
+date: 2019-08-29
+---
```

Comment:
Please replace the above comments here, otherwise the title will display an error.

Reply:
Sorry, I don't understand.

Reply:
How to do？